resources/assets/lib/chat-icon.tsx

```diff
@@ -49,7 +43,7 @@ export default class ChatIcon extends React.Component<Props> {
   }
 
   private unreadCountDisplay() {
-    if (this.props.worker.hasData) {
+    if (core.socketWorker.hasConnectedOnce) {
```

Comment:
this results in `...` turning into `0` before finally showing the correct number. Maybe it should still consult notification worker after all?